docs/user/tutorials/01-00-x-forwarded-for-header.md

```diff
@@ -1,132 +1,82 @@
-4. Select **Save**.
+  >[!WARNING]
+   > For production Deployments, it is strongly recommended to deploy Istio Ingress Gateway Pod to multiple nodes if you enable externalTrafficPolicy : Local. For more information, see Network Load Balancer.
+   >
+   >Default Istio installation profile configures PodAntiAffinity to ensure that Ingress Gateway Pods are evenly spread across all nodes. This guarantees that the above requirement is satisfied if your IngressGateway autoscaling configuration minReplicas is equal to or greater than the number of nodes. You can configure autoscaling options in the Istio CR using the field spec.config.components.ingressGateway.k8s.hpaSpec.minReplicas.
+   >[!TIP]
+   > If you use a GCP or Azure cluster, you can find your load balancer's IP address in the field status.loadBalancer.ingress of the ingress-gateway Service.
 <!-- tabs:end -->
 
-### Create a Workload for Verification
-
-1. [Create an HttpBin workload](https://kyma-project.io/#/api-gateway/user/tutorials/01-00-create-workload).
-2. Export the following values as environment variables.
-    ```bash
-    export DOMAIN_TO_EXPOSE_WORKLOADS={GATEWAY_DOMAIN}
-    export GATEWAY={GATEWAY_NAMESPACE}/GATEWAY_NAME}
-   ```
-3. Expose the HttpBin workload using a VirtualService.
-    ```bash
-    cat <<EOF | kubectl apply -f -
-    apiVersion: networking.istio.io/v1alpha3
-    kind: VirtualService
-    metadata:
-      name: httpbin
-      namespace: $NAMESPACE
-    spec:
-      hosts:
-      - "httpbin.$DOMAIN_TO_EXPOSE_WORKLOADS"
-      gateways:
-      - $GATEWAY
-      http:
-      - match:
-        - uri:
-            prefix: /
-        route:
-        - destination:
-            port:
-              number: 8000
-            host: httpbin.$NAMESPACE.svc.cluster.local
-    EOF
-    ```
-
-### Verify the X-Forwarded-For and X-Envoy-External-Address Headers
-1. Get your public IP address.
-    ```bash
-    curl -s https://api.ipify.org
-    ```
-
-2. Send a request to the HttpBin workload.
-    ```bash
-    curl -s "https://httpbin.$DOMAIN_TO_EXPOSE_WORKLOADS/get?show_env=true"
-    ```
-3. Verify that the response contains the **X-Forwarded-For** and **X-Envoy-External-Address** headers with your public IP address, for example:
-    ```json
-    {
-      "args": {
-        "show_env": "true"
-      },
-      "headers": {
-        "Accept": "...",
-        "Host": "...",
-        "User-Agent": "...",
-        "X-Envoy-Attempt-Count": "...",
-        "X-Envoy-External-Address": "165.1.187.197",
-        "X-Forwarded-Client-Cert": "...",
-        "X-Forwarded-For": "165.1.187.197",
-        "X-Forwarded-Proto": "...",
-        "X-Request-Id": "..."
-      },
-      "origin": "165.1.187.197",
-      "url": "..."
-    }
-    ``` 
+### Results
+When you call an exposed workload, the response contains the X-Forwarded-For and X-Envoy-External-Address headers with your public IP address. See an example response for the Client IP 165.1.187.197:
```

Comment:
```suggestion
When you call an exposed workload, the response contains the **X-Forwarded-For** and **X-Envoy-External-Address** headers with your public IP address. See an example response for the Client IP `165.1.187.197`:
```

Reply:
I definitely need some tool that will help copy all these formatting rules from markdown to XML and the other way around
Thanks for spotting all of these mistakes, I completely forgot to verify this

Reply:
No worries, that's what you have reviewers for :)